src/core/Footer.tsx

```diff
@@ -413,23 +413,23 @@ const Footer = ({ paths, urlBase, statusUrl }: FooterProps) => {
         <div className="col-span-full sm:col-span-1 inline-flex sm:ml-auto sm:items-center">
           <img
             className="mr-24 h-80"
-            src={paths.highestUserAdoption}
-            alt="Highest User Adoption 2023"
+            src={paths.bestSupport}
+            alt="Best Support 2025"
           />
           <img
             className="mr-24 h-80"
-            src={paths.bestSupport}
-            alt="Best Support 2023"
+            src={paths.highPerformer}
+            alt="High performer 2025"
```

Comment:
Can we have consistent capitalisation across all these alt attributes? Makes sense to have them be titleised, i.e. this one being "High Performer 2025"

Reply:
fixed